src/Pages/Nodes.razor

```diff
@@ -27,16 +27,25 @@
                 <h2>@(context.EditState) node</h2>
             </PopupTitleTemplate>
             <DataGridColumns>
-                <DataGridCommandColumn TItem="Node" Filterable="false" Width="120px">
+    
+                <DataGridCommandColumn TItem="Node" Filterable="false"  Editable="false" Width="60px">
+
+                    <EditCommandTemplate>
+                        <Dropdown @onclick:stopPropagation="true">
+                            <DropdownToggle Color="Color.Primary">
+                            </DropdownToggle>
+                            <DropdownMenu>
+                                <DropdownItem Clicked="context.Clicked">Edit</DropdownItem>
+                                <DropdownItem Clicked="@(() => ShowDeleteDialog(context.Item))">Delete</DropdownItem>
+                                <DropdownItem Clicked="@(() => DisabledNode(context.Item) )">@_NodesState(context.Item)</DropdownItem>
+                            </DropdownMenu>
+                        </Dropdown>
+                    </EditCommandTemplate>
+                    <DeleteCommandTemplate/>
```

Comment:
This line should be deleted right?

Reply:
No, I mean the delete command template tag. Because there is already a delete button up in the code.

Reply:
In this case we will have an extra delete button 
<img width="274" alt="image" src="https://github.com/Elenpay/NodeGuard/assets/81957232/7d28940b-ee2c-427e-bfb4-62914687e4d1">